### DISCUSSION TITLE
Member onboarding

### DISCUSSION THREAD

**Discussion by Sealjay** [Green-Software-Foundation/oc#11] [General]
I’ve had feedback that getting started after member onboarding isn’t as seamless as it could be.

For example, finding content, getting calendar invites for working groups, knowing which working groups align to interests, the best communication channels to use (email, GitHub, or we still have slack enabled), how knowledge is shared, how to complete a PR-FAQ, and how then to get calendar invites for individual projects.

What are your thoughts?

**Comment by jawache** [comment]
@Sealjay we did discuss this at the previous OC meeting but to summarize we have a 4 phase project for onboarding.

- Phase 1 - The digest emails that are being sent out every Monday now.
- Phase 2 - Better onboarding forms, a form for an employee to self register in the Foundation and an Form for a member to subscribe and be added to any of our WGs and Projects.
- Phase 3 - Measuring GitHub activity metrics for all members, so we can figure out who is active on which projects, which projects are active etc...
- Phase 4 - Monthly email report to each organization so they know (a) who from their org has joined (b) what wgs/projects are they subscribed to (c) how active they are.

We agreed once we are close to each phase @Oleg-Zhymolokhov would come in and demo the phase features.

**Reply by Sealjay** [reply]
That's fine, this discussion was raised on Feb 7th, and I remember chatting about it.

We haven't finalised the agenda yet - I wanted to talk with @tmcclell before we take it off the list, but I think we can remove it from the agenda after that.

**Comment by Sealjay** [comment]
Sharing this guide - on norms, etc: https://www.theopensourceway.org/

**Comment by tmcclell** [comment]
@Willmish - Brought up calendar are helpful for getting people onboarded. @seanmcilroy29 - Emails are adding to DL and then digest is forwarded to the DL. Conversation led into culture participation per different working groups. @jawache - repo on how we work together. Should we collect feedback during Asim's newsletter and @srini - what is working well. @NAMRATA-WOKE to generate draft and send to OC DL for feedback. @jawache - video? @Willmish - add to website. Agenda item for next meeting.

**Reply by Willmish** [reply]
If Szymon arrives here looking for the tag, it is supposed to be swapped for @Willmish :D
